scripts/vX_switch.sh

```diff
@@ -0,0 +1,30 @@
+usage() { 
+
+    echo './vX_switch.sh [vX]'
+    echo ''
+    echo '    vX = v2 or v3'
+    echo ''
+    echo 'Will change the symlinks on the ctp7 to point to the correct files for vX electronics. Ends with a call to recover.sh'
+    kill -INT $$
+}
+
+VX=$1
+
+if [ -z ${1+x} ]
+then
+    usage
+fi
+
+if [ $VX != "v2" ]  &&  [ $VX != "v3" ]
+then
+    usage
+fi   
+
+echo "Updating symlinks"
+ln -s /mnt/persistent/gemdaq_${VX}/ /mnt/persistent/gemuser/gemdaq_USER
```

Comment:
* remove the trailing `/`
* change to `ln -sn` so that it treats things as a directory and doesn't try to make a link inside a directory
  * probably also want to either `unlink` first, or `ln -sfn` to force the creation
* ~why is this pointing to `/mnt/persistent/gemuser/gemdaq_USER` instead of `/mnt/persistent/gemdaq_USER`?~

Reply:
fixed